cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocClient.java

```diff
@@ -628,7 +637,8 @@ void updateClusterLabels(String clusterName,
       Cluster cluster = getDataprocCluster(clusterName)
         .filter(c -> c.getStatus().getState() == ClusterStatus.State.RUNNING)
         .orElseThrow(() -> new DataprocRuntimeException("Dataproc cluster " + clusterName +
-                                                          " does not exist or not in running state"));
+                                                          " does not exist or not in running state",
+                ErrorGroup.CONFIGURATION, ErrorCodeProvider.UNKNOWN));
```

Comment:
this is not an unknown problem, we're trying to update labels for a cluster that doesn't exist, which is most likely a race condition, where it did exist before but got deleted by the time we reach here.

Reply:
Thanks, changed to CLUSTER_NOT_FOUND.